docs/docs/advanced/authentication.md

```diff
@@ -122,14 +125,18 @@ Handler middleware(Handler handler) {
       .use(requestLogger())
       .use(
         bearerTokenAuthentication<User>(
-          userFromToken: userRepository.fetchFromAccessToken,
-        ),
+          readUser: (context, token) {
+            final userRepository = context.read<UserRepository>();
+            return userRepository.fetchFromAccessToken(token);
+          }
+        ).build(),
```

Comment:
I would split the docs to a different pr to be merged after the package release

Reply:
Docs reverted and added a PR only for it!